realtimetrains.py

```diff
@@ -145,90 +158,58 @@ def populate(self):
             return False
         else:
             self.webpage_checksum = md5sum
-
-        soup = BeautifulSoup(r.text, "html.parser")
-        if soup.text == NO_SCHEDULE:
+        
+        if 'realtimeActivated' not in r.json().keys():
```

Comment:
`if blah not in dict` is the same as `if blah not in dict.keys()` and I think is a bit faster.